accepted/single-file/extract.md

```diff
@@ -78,11 +78,14 @@ For a single-file app, the extraction directory is `<base>/<app>/<bundle-id>`
 * `<base>` is 
 
   * `DOTNET_BUNDLE_EXTRACT_BASE_DIR` environment variable, if set.
-  * If not, defaults to 
-    * `%TEMP%\.net ` on Windows
-    * `$TMPDIR/.net` if `$TMPDIR` is set (Posix conforming OSes including Mac)
-    * Otherwise `/var/tmp/.net` (Ubuntu)  if the directory exists.
-    * Otherwise  `/tmp/.net` 
+  * If not, on Windows, defaults to `%TEMP%\.net`
+  * On Unix-like systems, where multiple users may use a single system, an approach that removes the possibility of name collisions and other users creating files to prevent an application to start (by a malicious user creating a predictable directory name) is used instead:
+     * The directory to extract the bundle is created with `mkdtemp()`, using the `$TMPDIR` environment variable (if set), `/var/tmp/` (if exists, because it survives reboots), falling back to `/tmp` (does not survive reboots, it's often a ramdisk) in the template (e.g. `/var/tmp/dotnet-<app>-XXXXXX`);
+     * Symbolic link is created in `~/.cache/dotnet/<app>/<bundle-id>` pointing to the directory created by `mkdtemp()`;
+         * `~/.cache` might be in another directory if `$XDG_CACHE_HOME` is set. See [the XDG spec for information](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html).
```

Comment:
It would be good to write this as a declarative spec. So, I'd write something like:
If `$XDG_CACHE_HOME` is set, create a symbolic link at `$XDG_CACHE_HOME/.net`
Otherwise  `~/.cache/.net`


So, replace "`~/.cache/` might be " to "is".

Reply:
Done